test/unit/discovery_attribute_set_test.rb

```diff
@@ -1,4 +1,5 @@
-require 'test_helper'
```

Comment:
Actually, https://github.com/theforeman/foreman_plugin_template doesn't have a `test/test_helper.rb`at all. So this should be possible.

How about doing this in a separate PR to get the actual fix in fast?